### ISSUE TITLE
Does markdeck run on Windows?

### ISSUE THREAD

**Issue by n179911** [arnehilmann/markdeck#8]
I have tried to install markdeck on windows.
I install curel and bash and ubuntu subsystem and docker on windows.
I run the curl command and then start 'bash', i run the 'scaffold'
But it cant' find 'docker'.

How can I fix this and get this to work in Windows?

**Comment by arnehilmann** [comment]
Unfortunately I cannot get docker running in a windows VM right now (sometimes it works, sometimes not *grr*) So I am guessing here right now.

Did you install docker...
1) by unpacking a zip file (via download.docker.com, wrong way I guess), or 
2) by executing an installer.exe? (via store. docker.com, correct way)?

I tried both, and with variant 1) I had the same problem. 
With variant 2), while not being able to start the docker daemon, at least I could find and use the `docker` command.

I will try to get docker running on Windows again...

**Comment by mcab** [comment]
I don't believe Windows Subsystem for Linux (WSL) supports running Docker containers, so you'll have to have Docker installed on the Windows host, and then expose it for WSL to use. [This guide](https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly) might help; I might experiment with getting it to run to see if markdeck will run on Windows.

**Comment by n179911** [comment]
I tried the method #2.

Thanks. I will take a look at the above guide.

**Comment by mcab** [comment]
After further testing, WSL allows for the convenience scripts to help stop/start/update the project.

If you're comfortable with Docker, you can follow what [scaffold](https://github.com/arnehilmann/markdeck/blob/bf654b83ecb57ce678ec4b15669b645b8a266ec1/scaffold) does, and just spin up the Docker container manually.

It should launch, and you can can edit your slides from there.

Downside is that you don't have built-in support for updating/starting, but that should just be reliant on changes to `docker-compose.yaml`.

**Comment by dmastag** [comment]
I am using docker toolbox on windows 10
And am getting this error

```
$ ./markdeck
./markdeck: line 46: USER: unbound variable
```


Any idea what I should do ?

**Comment by arnehilmann** [comment]
Do you use WSL?

**Comment by dmastag** [comment]
Unfortunately cannot use WSL on my Workstation

**Comment by dmastag** [comment]
So I tried using WSL and it was not working as I could not properly install Docker Toolbox and have it connect with WSL
For some reason Oracle Virtual Box (used for Docker Toolbox) seems to mess up WSL. So I can only have one up at a time.

In the end I just used a regular Ubuntu Virtual Box which worked.
So I assume that I could also use Vagrant.

**Comment by arnehilmann** [comment]
I tested it on a windows 10 box with "git for windows" (and its bash emulator), and voila: markdeck 0.54.3 worked!
